### DISCUSSION TITLE
請問哪裡有這個包

### DISCUSSION THREAD

**Discussion by Chia-tau** [cotaku/Bibi-Release#87] [Q&A]
Microsoft.NET.CoreRuntime.1.1.Appx
![IMG_20240405_171356.jpg](https://github.com/cotaku/Bibi-Release/assets/162953601/128f7604-7300-44c1-a881-7494c295c773)

不然無法安裝啊啊啊

**Comment by cotaku** [comment]
https://github.com/cotaku/Bibi-Release/issues/71#issuecomment-1970363248

**Reply by Chia-tau** [reply]
請問應該下載哪個包如果是Windows 10 Mobile的話![Screenshot_20240406_025958.jpg](https://github.com/cotaku/Bibi-Release/assets/162953601/c242f5f9-a118-4462-b6f6-72ec9cc8797c)

**Reply by cotaku** [reply]
手机的话直接商店安装吧：
https://github.com/cotaku/Bibi-Release/discussions/61
